lib/core/versions/latest/Did.ts

```diff
@@ -62,19 +62,43 @@ export default class Did {
    * @param didString Short or long-form DID string.
    */
   public static async create (didString: string, didMethodName: string): Promise<Did> {
+    const methodNameParts = didMethodName.split(':');
```

Comment:
I think we need to file an issue to support/disambiguate "method network suffix" from method name.

Reply:
https://github.com/decentralized-identity/sidetree/issues/470